ncard_app/templates/events/dashboard.html

```diff
@@ -1,32 +1,63 @@
 {% extends 'events/base.html' %}
+{% load static %}
 
-{% block title %}NCARD Database{% endblock %}
-
+{% block title %}Dashboard | NCARD{% endblock %}
 
 {% block content%}
-    <div class="container-fluid">
-        <div class="text-center row">
-            <div class="col-sm">
-                <a href="{% url 'list-people' %}">
-                    <button type="button" class="btn btn-outline-info btn-lg">People</button>
-                </a>
-            </div>
-            <div class="col-sm">
-                <a href="{% url 'list-organisations' %}">
-                    <button type="button" class="btn btn-outline-info btn-lg">Organisation</button>
-                </a>
-            </div>
-            <div class="col-sm">
-                <a href="#">
-                    <button type="button" class="btn btn-outline-info btn-lg">Button 3</button>
-                </a>
+    <div class="row mt-3">
+        <div class="text-center">
+            <h1 class="ncard-dark-green" >NCARD Dashboard</h1>
+        </div>
+    </div>
+
+    <div class="container" style="text-align:center;">
+        <div class="dashboard" style="padding:50px; margin:50px;">
+            <div class="row">
+                <div class="col-md-3">
+                    <a href="{% url 'list-awards' %}">
+                        <img class="icon" src="{% static 'awards-icon.png' %}" alt="awards icon">
```

Comment:
I can't find any of these icon PNG files in your pull request. Make sure to add them. Also, what are the license terms for these icons?